scripts/store/main/reducer.ts

```diff
@@ -57,8 +57,13 @@ export default function (state = initialState, action: ActionTypes): SessionStat
             break;
         }
         case Constants.ADD_TO_BASKET: {
-            showToastDialog();
-            action.payload.count == 1 ? callVibrate(VIBRATION_ADD_TIME) : callVibrate(VIBRATION_REMOVE_TIME);
+            if (action.payload.count == -1) {
```

Comment:
We should write three equals for exact results :)

Reply:
DONE.